contracts/HintHelpers.sol

```diff
@@ -202,6 +209,33 @@ contract HintHelpers is VestaBase, CheckContract {
 		}
 	}
 
+	/* getLiquidatableAmount(address _asset) - returns total amount of VST liquidatable from an asset.
+    
+    This function goes thru vault from most at risk to least and counts the amount of VST liquidatable until it hits a vault
+	that is not liquidatable.
+    */
+	function getLiquidatableAmount(address _asset) external view returns (uint256 result_) {
+		uint256 price = priceFeed.fetchPrice(_asset);
```

Comment:
externalPrice(_asset). fetchPrice is a write function. You might need to update the IPriceFeed for the one from [OraclePriceFetcher](https://github.com/vesta-finance/OraclePriceFetcher/blob/master/src/main/interfaces/IPriceFeedV2.sol)

Reply:
current IPriceFeed also has it as write function so i think its fine.

Reply:
I'll update it before deploying, but your function is a viewable one. So your compiler is supposed to report this as an error since you can't write in a viewable function